### ISSUE TITLE
Comfort [BUG]

### ISSUE THREAD

**Issue by paohira** [valheimPlus/ValheimPlus#784]
OS: Windows 10 Client
    Valheim Game Version: 0.212.9
    V+ Mod Version: 0.9, 0.9.11

**Describe the bug**

The maximum comfort level reacheable seems to be 18 instead of 23

**To Reproduce**

Steps for the devs to reproduce the behavior:

It looks like the carpets doesn't give you +1 comfort each, just the first one gives you +1 confort

**Expected behavior**

**Troubleshooting**

What steps have you taken to try to fix?

**Screenshots**

here's the chosen config  https://pastebin.com/M8SwiX4m

**Additional context**

no other mod installed

**Comment by Grantapher** [comment]
From [Valheim patch notes 0.212.9 (Dec 20, 2022)](https://store.steampowered.com/news/app/892970/view/5201125680700004826):
> Comfort from different carpets no longer stacks

**Comment by kiloforce** [comment]
Is this a V+ issue, or a base game issue?

**Comment by paohira** [comment]
I saw video of mistland patch that reach comfort 23

https://www.youtube.com/watch?v=HGn7ic3lyVg

But honestly I didn't try without the V+

**Comment by SSimp258** [comment]
> I saw video of mistland patch that reach comfort 23
> 
> https://www.youtube.com/watch?v=HGn7ic3lyVg
> 
> But honestly I didn't try without the V+

There's a note on that video you linked:

![image](https://user-images.githubusercontent.com/96361739/218317044-9fb201a9-7232-472b-85d7-ec303301a95e.png)

**Comment by paohira** [comment]
oh.. so the maximum level is actualy 18?

**Comment by daerSeebaer** [comment]
> oh.. so the maximum level is actualy 18?

Yes, it is. Can you close this issue, since it is not V+ related?
